src/pages/MedicationPage.js

```diff
@@ -69,31 +90,27 @@ export default function MedicationPage({
     </Grid>
   )
 
+  function handleDayClick(day) {
+    setSelectedDay(day)
+  }
+
   function convertToMinutes(time) {
     const timeArray = time.split(':')
     const minutes = timeArray[0] * 60 + timeArray[1]
     return Number(minutes)
   }
 
-  function handleDeleteClick(id) {
-    const index = medications.findIndex(medication => medication.id === id)
-    setMedications([
-      ...medications.slice(0, index),
-      ...medications.slice(index + 1),
-    ])
-  }
-
   function handleEditClick(id) {
-    const index = medications.findIndex(medication => medication.id === id)
-    setMedicationToEdit(medications[index])
+    setMedicationToEditId(id)
     setActivePage('form')
   }
 }
 const Grid = styled.div`
```

Comment:
Please have a look if not a section is the right element.

Reply:
the grid contains the whole page, header included, so basically the whole body. in my understanding a div is the right element here.